views/company-number.html

```diff
@@ -0,0 +1,63 @@
+{% extends "layout.html" %}
+{% block pageTitle %}
+  What is the company number
+{% endblock %}
+
+{% block backLink %}
+  {{ govukBackLink({
+  href: "/objecting-entity-contact-details",
+  text: "Back"
+}) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form action="/company-number" method="post">
+        {% if errorNum == true %}
+          {{ govukErrorSummary({
+            titleText: "There is a problem",
+            errorList: errorList
+          }) }}
+          {{ govukInput({
+            label: {
+              text: "What is the company number?",
+              isPageHeading: true,
+              classes: "govuk-label--xl"
+            },
+            hint: {
+              text: "This is the 8 character reference issued by Companies House when the company was set up."
+            },
+            classes: "govuk-input--company-number",
+            id: "company-number",
+            name: "company-number",
+            errorMessage: {
+              text: "Enter the company number"
+            }
+          }) }}
+        {% else %}
+          {{ govukInput({
```

Comment:
can you look at PTF company-number.html. We can set the error message in it's own variable and this will remove the govukInput duplication

Reply:
Done. Although note that error handling and use of this variable will come with implementation of a later story.